### PR TITLE
Dfr 379 list of hearing consent case

### PR DESCRIPTION
**Pull request by rishikrsharma**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFR-379

### Change description ###
List for hearing for consented case type

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -1426,5 +1426,12 @@
     "CaseEventID": "FR_updateOrganisation",
     "UserRole": "caseworker-divorce-systemupdate",
     "CRUD": "R"
+  },
```
Comment (EllisD-B) [inline definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json:1429]: Do solicitors and judiciary not require read access?

File: definitions/consented/json/AuthorisationCaseField/AuthorisationCaseField.json [hunk 2]
```diff
@@ -6108,5 +6108,12 @@
     "CaseFieldID": "otherVariationCollection",
     "UserRole": "caseworker-divorce-financialremedy-judiciary",
     "CRUD": "CURD"
+  },
```
Comment (EllisD-B) [inline definitions/consented/json/AuthorisationCaseField/AuthorisationCaseField.json:6111]: ^^

**Review by EllisD-B — COMMENTED** [review]
small comment re auths

File: definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json (hunk 1 above)
Comment (rishikrsharma) [reply definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json:1429]: its only for caseworker

Comment (rishikrsharma) [reply definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json:1429]: addd read only accesss for solicitors and judiciary

Comment (rishikrsharma) [reply definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json:1429]: cheers

**Review by EllisD-B — APPROVED** [review]

**Pull request merged**